### DISCUSSION TITLE
Create a migration tool

### DISCUSSION THREAD

**Discussion by upintheairsheep** [IviriusCommunity/Rebound#102] [Ideas]
Hello, the Windows 11 experience is missing one thing: a proper migration tool, we haven't had a proper first-party tool to migrate data from one target to another since Windows 8.1. The Windows Backup tool is very bad, as the point of migration for power users is 98% of the time to set up a computer after upgrading is reinstalling apps and reconfiguring settings, not moving files. You should create a tool based on https://github.com/builtbybel/Appcopier to migrate to a new PC and replace Windows Easy Transfer (Windows Easy Transfer was a USMT GUI for end users), and in the future, be able to migrate from (and possibly to in the future) macOS, ChromeOS Flex, and Linux (as well as Android and iPadOS on tablets). In the tool, a link to PCMover and ZInstall should be added, as it is a tool that can migrate programs without reinstallation (for people who rely on a program with a lost installer, or has a large amount of rare installed programs), not as an advertisement, but because no open source tool can reliably do this. MS themselves has a chance of adopting the tool if it can transfer from other platforms.

**Comment by IviriusMain** [comment]
Sounds like Samsung Smart Switch

**Reply by Lamparter** [reply]
God have you seen the videos for that app?

**Reply by IviriusMain** [reply]
I used it today

**Reply by upintheairsheep** [reply]
Yes, it is basically it, they also have “Galaxy Book Smart Switch” but it’s a closed source (of course) Windows Store app with artificial restrictions to make it only for Samsung devices (as expected)

**Reply by IviriusMain** [reply]
Samsung ecosystem moment
Anyway this app would be a very good but also complicated project and honestly deserves its own software development team so I'm personally gonna pass on the offer
